json/fr.json

```diff
@@ -77,10 +77,10 @@
     "Leave Team" : "Quitter l'équipe",
     "Leave" : "Quitter",
     "Login": "Connexion",
-    "Logout Other Browser Sessions" : "Déconnecter les sessions ouvertes sur d'autres navigateurs",
-    "Logout": "Déconnexion",
+    "Log Out Other Browser Sessions" : "Déconnecter les sessions ouvertes sur d'autres navigateurs",
+    "Log Out": "Déconnexion",
     "Manage Account" : "Gérer le compte",
-    "Manage and logout your active sessions on other browsers and devices." : "Gérer et déconnecter vos sessions actives sur les autres navigateurs et appareils.",
```

Comment:
Keys used in Laravel products cannot be deleted.

See: https://github.com/Laravel-Lang/lang/blob/master/script/en/en.json#L86-L87